### DISCUSSION TITLE
Verified publishers

### DISCUSSION THREAD

**Discussion by isidorn** [microsoft/vscode-discussions#29] [Announcements]
Extension authors can become verified by verifying ownership of an [eligible domain](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#eligible-domains) associated with your brand or identity. Verifying your publisher will demonstrate your authenticity and add a verified badge to your extensions.

More details in our docs here https://code.visualstudio.com/api/working-with-extensions/publishing-extension#verify-a-publisher

Try it out and let us know how it goes.

![image](https://user-images.githubusercontent.com/1926584/172636695-40e0fb5e-f56a-43e9-a334-1c9f79c42548.png)

**Comment by PEZ** [comment]
Thanks for informing about this!

It took me a while to figure out where to find the Verified Domain field. Because the instructions say ”Create or select the publisher”, and there was no publisher to select. It felt wrong to create one. But then I checked the **Details** tab, and there I found the field.

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/30010/172712943-5ef34af7-304d-4b32-b59a-5e2b728ea540.png">

Not sure if this is how it looks like for others, but anyway.

**Comment by busticated** [comment]
just ran into another weird UX thing. once i entered my domain into the "Domain name" field, i had to click the "save" button up top before the "verify" button became active 🤷‍♂️

**Comment by isidorn** [comment]
@pez @busticated good feedback. Yes the UI deserves some polish. 

@chgill-MSFT can you pass this feedback along to the MP dev. Or create new issues in the MP repo so we track this?

**Comment by PEZ** [comment]
It seems the verification process for my organisation has gotten stuck or lost somewhere.

<img width="948" alt="image" src="https://user-images.githubusercontent.com/30010/200166492-f656cd86-e0b2-44a5-bae6-bd5fa2fcf57d.png">

According to this discussion, I submitted ut June 8, and I am guessing it is supposed to be a tad quicker than it has been. 😄

**Reply by isidorn** [reply]
Thanks for reaching out and sorry about that.
@prashantvc can you check what is the status here? Do we know the total number of authors that started verification but the process never finished?
fyi @anangaur

**Reply by bwateratmsft** [reply]
I'm seeing the same as @PEZ.

**Reply by prashantvc** [reply]
I don’t have number top of head, but it should be small.
@PEZ @bwateratmsft can you email vsmarketplace@microsoft.com, I will have someone to look at it

**Reply by isidorn** [reply]
@prashantvc let's double check that not a lot of users are blocked on this.

**Reply by PEZ** [reply]
My publisher is verified now. Thanks! 🙏

**Reply by bwateratmsft** [reply]
Mine as well. Thanks @prashantvc and @isidorn!
